docs/source/release/v6.2.0/mantidworkbench.rst

```diff
@@ -24,6 +24,7 @@ New and Improved
 - Script editor tab completion and call tip support for Numpy 1.21
 - The visibility of a component parameter in the Pick tab of the InstrumentViewer is now steered by the 'visible' atrribute of a parameter in IPF
 - Plot legends can be shown or hidden from the plot context menu.
+- ADS signal handlers are made synchronized in `WorkspaceSelector`. This reduces the probability of hard crash when interacting with the widget while a script manipulating a large number of workspaces is being run.
```

Comment:
I think this should be 'ADS signal handlers are now synchronized in the `WorkspaceSelector`'